src/libs/Navigation/AppNavigator/createCustomBottomTabNavigator/TopBar.tsx

```diff
@@ -63,12 +61,12 @@ function TopBar({breadcrumbLabel, activeWorkspaceID, shouldDisplaySearch = true,
                     </View>
                 </View>
                 {displaySignIn && <SignInButton />}
-                {shouldDisplayCancelSearch && (
+                {shouldDisplayCancel && (
```

Comment:
NAB - reminder for me to discuss 1 thing about using shouldDisplayCancel + callback